src/savegame_legacy.cpp

```diff
@@ -1907,7 +1907,7 @@ void item::load_legacy(std::stringstream & dump) {
         }
     }
 
-    dump >> burnt >> poison >> ammotmp >> owned >> bday >>
+    dump >> burnt >> poison >> ammotmp >> bday >>
```

Comment:
Now the value of owned that was in the file will be put into `bday`, and all the other values behind this are out of order.